### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_CreateDatabaseMasterKey_HighContrast:Color contrast ratio for "Preview button"is below the expected ratio i.e 7:1 when using high contrast with white theme.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5781]
[Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
 GitHubTags: #A11yMAS; #A11yTCS; #MAS1.4.3; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:** 
Application name: Azure Data Studio 
Version: 1.8.0-insider (user setup)
 Electron: 3.1.8
 Node.js: 10.2.0
 V8: 6.6.346.32
 OS: Windows_NT x64 10.0.17763

**Additional Details**
Tool Name: HighContrast
MAS Reference:MAS1.4.3

**Repro Steps:**  
1.Launch Visual Studio code desktop application.
2.Navigate to "Enable SQL server2019 preview->select table from the database->Create External Table screen should open->click on next button.
3.Navigate to Setting button of Activity Bar and select "Color Theme" option.
4.Open color contrast analyser and check the color contrast ratio.
5.It shows color contrast ratio less than expected color contrast ratio.

**Actual:**
After applying High Contrast white theme to "Preview", "Next"and "Cancel" buttons it shows color contrast ratio as  "3.458:1"

**Same issue observed for the below pages:**
**1.CreateConntionDataSrc:** Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button.
**2.CSVModifyColumn:** Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button->Click on next button.
**3.CSVSelectDestinationDatbse :** Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV files
**4.MapDataSrcObjects :** Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button->fill all the required details->click on next button.

**Expected:**
The expected color contrast ratio for "Preview", "Next" and "Cancel" buttons should be greater than or equals to 7:1 ratio.

**User Impact:**
Low vision User will find it difficult to understand and will be confused if they will not able to see the buttons on this page.

**Recommendations:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference:**
MAS1.4.3- https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Ba73546c7-765f-489c-b18f-afb659fe99e6%7D

**Attachment For Reference:**
![whtcontrastpwd](https://user-images.githubusercontent.com/39204909/58682892-e52cef00-838f-11e9-8cb2-3cdede318260.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by udeeshagautam** [comment]
@AccessibilityTestingTeam-TCS please let us know what you mean by "high contrast with white theme" - the image seems like default white theme. do you mean default theme?

![image](https://user-images.githubusercontent.com/46980425/66953488-7c2ed680-f013-11e9-85b3-1d7f36a48ce3.png)

If so this should be fixed for default theme.

**Comment by udeeshagautam** [comment]
@AccessibilityTestingTeam-TCS  please note that we have a high contrast setting. Please try thebehavior on the same for high contrast scenarios.

**Comment by alanrenmsft** [comment]
for default light theme, we have made changes a while back to the, now the contrast ratio is 4.51 which meets the requirement of 4.5.
![Screen Shot 2019-11-07 at 1 44 13 PM](https://user-images.githubusercontent.com/13777222/68430479-f7902d80-0164-11ea-8b96-2f8552b5bc48.png)
![Screen Shot 2019-11-07 at 1 44 37 PM](https://user-images.githubusercontent.com/13777222/68430480-f7902d80-0164-11ea-9d8b-e3b0ed91123c.png)


for high contrast mode, it is white and black

![Screen Shot 2019-11-07 at 1 43 45 PM](https://user-images.githubusercontent.com/13777222/68430510-0a0a6700-0165-11ea-8e59-c6249738e338.png)

closing as fixed.
